### PR TITLE
Add instrument driver for oxford kelvinox.

### PR DESCRIPTION
**Pull request by CJvanDiepen**
Add instrument driver for Oxford Kelvinox IGH.

Changes proposed in this pull request:
    Add instrument driver

@jenshnielsen

### CODE REVIEW

File: qcodes/instrument_drivers/oxford/kelvinox.py [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+                           get_cmd=self._get_still_status)
+        self.add_parameter('sorb_status',
+                           get_cmd=self._get_sorb_status)
+        self.add_parameter('still_power',
+                           unit='mW',
+                           get_cmd=self._get_still_power,
+                           set_cmd=self._set_still_power)
+        self.add_parameter('sorb_temp',
+                           unit='K',
+                           get_cmd=self._get_still_power,
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: This looks like the wrong cmd, same below for the set cmd

File: qcodes/instrument_drivers/oxford/kelvinox.py [hunk 2]
```diff
@@ -0,0 +1,534 @@
+# OxfordInstruments_Kelvinox_IGH.py class, to perform the communication between the Wrapper and the device
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: OxfordInstruments_Kelvinox_IGH.py to either OxfordInstruments_Kelvinox_IGH or kelvinox.py

File: qcodes/instrument_drivers/oxford/kelvinox.py [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+    def get_idn(self):
+        """
+        Overides the function of Instrument since IPS120 does not support '*IDN?'
+
+        This string is supposed to be a comma-separated list of vendor, model,
+        serial, and firmware, but semicolon and colon are also common
+        separators so we accept them here as well.
+
+        Returns:
+            A dict containing vendor, model, serial, and firmware.
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: This produces a warning in the docs build 

/home/travis/build/QCoDeS/Qcodes/qcodes/instrument_drivers/oxford/kelvinox.py:docstring of qcodes.instrument_drivers.oxford.kelvinox.OxfordInstruments_Kelvinox_IGH.get_idn:1: WARNING: Inline emphasis start-string without end-string.

It's better to write it as ```  `*IDN?` ``` to escape the ```*```

File: qcodes/instrument_drivers/oxford/kelvinox.py [hunk 4]
```diff
@@ -0,0 +1,534 @@
...
+                           unit='mbar',
+                           get_cmd=self._get_G1)
+        self.add_parameter('V6_valve',
+                           unit='%',
+                           get_cmd=self._get_V6_valve,
+                           set_cmd=self._set_V6_valve)
+        self.add_parameter('V12A_valve',
+                           unit='%',
+                           get_cmd=self._get_V6_valve,
+                           set_cmd=self._set_V6_valve)
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: Wrong cmd?

File: qcodes/instrument_drivers/oxford/kelvinox.py [hunk 5]
```diff
@@ -0,0 +1,534 @@
...
+        This string is supposed to be a comma-separated list of vendor, model,
+        serial, and firmware, but semicolon and colon are also common
+        separators so we accept them here as well.
+
+        Returns:
+            A dict containing vendor, model, serial, and firmware.
+        """
+        idparts = ['Oxford Instruments', 'Kelvinox IGH', None, None]
+
+        return dict(zip(('vendor', 'model', 'serial', 'firmware'), idparts))
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: There is no good way to get the firmware/serial number?

File: qcodes/instrument_drivers/oxford/kelvinox.py [hunk 6]
```diff
@@ -0,0 +1,534 @@
...
+            O1 : Still ON, sorb OFF
+            O2 : Still OFF, sorb on T control
+            O3 : Still ON, sorb ON T control
+            O4 : Still OFF, sorb on power control
+            O3 : Still ON, sorb ON power control
+        """
+        self.remote()
+        log.info('Set still and sorb status')
+        self._execute(status)
+        self.local()
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: What happens if it was in remote mode before executing this command?

Comment (CJvanDiepen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: This is the way it was operated before and we don't have a reason to change it, so I think it is best to leave it like it is.

**Comment by CJvanDiepen** [comment]
@jenshnielsen Please take another look! If you are happy with it then I am too ;)

File: qcodes/instrument_drivers/oxford/kelvinox.py (hunk 6 above)
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/kelvinox.py]: Ok cool

**Pull request merged**